eva_submission/eload_utils.py

```diff
@@ -153,6 +154,10 @@ def download_file(url, dest):
     urllib.request.urlcleanup()
 
 
+def check_project_format(project_accession):
+    return re.match(r'^PRJ(EB|NA)', project_accession)
```

Comment:
Would be good to reuse this in other places (I think just [here](https://github.com/EBIvariation/eva-submission/blob/b99c3659a6a60a0898efe2dd8a8096f9b38e7d19/eva_submission/ENA_submission/xlsx_to_ENA_xml.py#L106) actually)